scripts/yul_coverage.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -e
+
+ROOT_DIR="$(dirname "$0")"/..
+
+FAILED=()
+SUCCESS=()
+test_file()
+{
+  local SOL_FILE
+  local OUTPUT
+
+  SOL_FILE=${1}
+  OUTPUT=$(solc --ir "${SOL_FILE}" 2>&1 || true)
+  if [[ ${OUTPUT} == *"UnimplementedFeatureError"* ]]; then
+    FAILED+=("${SOL_FILE}")
+  else
+    SUCCESS+=("${SOL_FILE}")
+  fi
```

Comment:
There are a lot of  of `InternalCompilerError`s that fall into the `SUCCESS` branch here. For example:
```
Internal compiler error during compilation: /solidity/libsolidity/codegen/ir/IRGeneratorForStatements.cpp(956): Throw in function virtual void solidity::frontend::IRGeneratorForStatements::endVisit(const solidity::frontend::MemberAccess&) Dynamic exception type: boost::wrapexcept<solidity::langutil::InternalCompilerError> std::exception::what: Member access to unknown type. [solidity::util::tag_comment*] = Member access to unknown type.
```

Is that intended?

Reply:
nope that was not intended like this ;)